code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -256,6 +294,142 @@
 	path = /obj/item/clothing/accessory/badge/idbadge/intel
 
 /datum/gear/accessory/sleeve_patch
-	display_name = "shoulder sleeve patch"
+	display_name = "shoulder sleeve patch (colourable)"
 	path = /obj/item/clothing/accessory/sleevepatch
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/namepin
+	display_name = "pin tag (colourable)"
+	path = /obj/item/clothing/accessory/badge/namepin
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/asym
+	display_name = "asymmetric jacket (selection)"
+	path = /obj/item/clothing/accessory/asymmetric
+
+/datum/gear/accessory/asym/New()
+	..()
+	var/asyms = list()
+	asyms["blue asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric
+	asyms["purple asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric/purple
+	asyms["green asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric/green
+	gear_tweaks += new/datum/gear_tweak/path(asyms)
+
+/datum/gear/accessory/sash
+	display_name = "sash (colourable)"
+	path = /obj/item/clothing/accessory/sash
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/cowledvest
+	display_name = "cowled vest"
+	path = /obj/item/clothing/accessory/cowled_vest
+
+/datum/gear/accessory/halfcape
+	display_name = "half cape"
+	path = /obj/item/clothing/accessory/halfcape
+
+/datum/gear/accessory/fullcape
+	display_name = "full cape"
```

Comment:
No capes please. We removed cloaks a while ago due to their conflict with the station and the setting's aesthetics.